### PR TITLE
implement HTTPS for staging.binder.pangeo.io

### PR DESCRIPTION
**Pull request by dsludwig**
kube-lego is tedious and inscrutable... it fails until it doesn't.

But this looks to be working now!

### CODE REVIEW

File: helm-chart/pangeo-binder/values.yaml [hunk 1]
```diff
@@ -2,6 +2,58 @@ binderhub:
...
+nginx-ingress:
+  rbac:
+    create: true
+  defaultBackend:
+    minAvailable: 0
+  # statsExporter:
+  #   service:
+  #     annotations:
+  #       prometheus.io/scrape: 'true'
+  #       prometheus.io/port: '10254'
```
Comment (jhamman) [inline helm-chart/pangeo-binder/values.yaml]: will we need to turn this on for the full deployment? What would happen if we uncommented these lines?

**Review by jhamman — APPROVED** [review]
Looks good to me. Thanks for putting this together. Hopefully cert-manager is easier when we move there.

**Comment by jhamman** [comment]
Also, I did manage to run some tests on the staging deployment tonight and everything seems to be in full working order.

File: helm-chart/pangeo-binder/values.yaml (hunk 1 above)
Comment (dsludwig) [reply helm-chart/pangeo-binder/values.yaml]: I don't think we have the stats service set up to scrape these stats. I don't think there's any harm in turning this on, but I don't think we'll see anything from it either.

**Comment by guillaumeeb** [comment]
Many thanks for that work @dsludwig and @jhamman. This works great from my corporate network!

That's another issue, but I'm a bit lost bewteen all the pangeo.io website : dev, staging.binder, binder. Maybe this should be clarified somewere? Or is it already?

**Pull request merged**